src/Tile/RadioTile.svelte

```diff
@@ -8,6 +8,9 @@
   /** Specify the value of the radio input */
   export let value = "";
 
+  /** Specify the title of the selectable tile */
+  export let title = "title";
```

Comment:
What's this prop for?

Reply:
aligning with your SelectableTile

Reply:
Remove

Reply:
Ok actioned, but why so hard on the prop!?